src/pkg/packaging/windows/package.targets

```
@@ -61,16 +61,19 @@
         <HostMsi>$(SharedHostInstallerFile)</HostMsi>
         <HostFxrMsi>$(HostFxrInstallerFile)</HostFxrMsi>
         <SharedBundle>$(CombinedInstallerFile)</SharedBundle>
+       <!-- Warning: changing the value "BundleInstallerOutputGuidString" can only occur on a product-band boundary [ProductBandVersion], 
+               Changing "BundleInstallerOutputGuidString" mid-product-band will break the upgradability of the Shared Framework bundle installer. -->
+        <BundleInstallerOutputGuidString>Dotnet Shared Framework Bundle Installer $(ProductBandVersion) $(PackageTargetRid)</BundleInstallerOutputGuidString>
```

Minor change to be a little more consistent:

```suggestion
        <BundleInstallerOutputGuidString>.NET Core Shared Framework Bundle Installer $(ProductBandVersion) $(PackageTargetRid)</BundleInstallerOutputGuidString>
```

Doesn't matter, but might as well make it a little better while we can change it. 😄